hat/assets/js/apps/Iaso/domains/orgUnits/components/TreeView/OrgUnitTreeviewModal.tsx

```diff
@@ -231,83 +275,59 @@ const OrgUnitTreeviewModal = ({
             >
                 <SettingsPopper setSettings={setSettings} settings={settings} />
             </Box>
-            <Box mt={1}>
-                <TreeViewWithSearch
-                    getChildrenData={id =>
-                        getChildrenData(id, validationStatus)
-                    }
-                    getRootData={getRootDataWithSource}
-                    label={makeTreeviewLabel(
-                        classes,
-                        showStatusIconInTree,
-                        displayTypes,
-                    )}
-                    toggleOnLabelClick={toggleOnLabelClick}
-                    onSelect={onOrgUnitSelect}
-                    request={searchOrgUnitsWithSource}
-                    makeDropDownText={orgUnit => (
-                        <OrgUnitLabel orgUnit={orgUnit} />
-                    )}
-                    toolTip={tooltip}
-                    parseNodeIds={getOrgUnitAncestors}
-                    multiselect={multiselect}
-                    queryOptions={{ keepPreviousData: true }}
-                    childrenQueryOptions={{ keepPreviousData: true }}
-                    preselected={selectedOrgUnitsIds}
-                    preexpanded={selectedOrgUnitParents}
-                    selectedData={selectedOrgUnits}
-                    onUpdate={onUpdate}
-                    allowSelection={item => {
-                        if (allowedTypes.length === 0) return true;
-                        return allowedTypes.includes(item.org_unit_type_id);
-                    }}
-                    dependency={validationStatus}
-                    childrenDependency={validationStatus}
-                />
+            <Box position="relative">
+                {isFetchingOrgUnit && <LoadingSpinner absolute />}
+                <Box mt={1}>
+                    <TreeViewWithSearch
+                        getChildrenData={id =>
+                            getChildrenData(id, statusSettings)
+                        }
+                        getRootData={getRootDataWithSource}
+                        label={(orgUnit: OrgUnit) => (
+                            <TreeViewLabel
+                                orgUnit={orgUnit}
+                                withStatusIcon={showStatusIconInTree}
+                                withType={displayTypes}
+                            />
+                        )}
+                        toggleOnLabelClick={toggleOnLabelClick}
+                        onSelect={onOrgUnitSelect}
+                        request={searchOrgUnitsWithSource}
+                        makeDropDownText={orgUnit => (
+                            <OrgUnitLabel orgUnit={orgUnit} />
+                        )}
```

Comment:
This doesn't need to be a component. The `DynamicSelect` in bluesquare-components just puts its return value in a `Typography`.
 (es-lint complains that we pass a component that way)